lib/parser.js

```diff
@@ -174,15 +174,8 @@ module.exports = class Parser {
         }
       }
 
-      if (!this.options.loose) {
-        if (this.nextToken[0] === 'word') {
-          return this.word();
-        }
-      }
-      else {
-        if ((!this.current.nodes.length || (this.current.last && this.current.last.type === 'operator')) && this.nextToken[0] === 'word') {
-          return this.word();
-        }
+      if (this.nextToken[0] === 'word') {
```

Comment:
so why don't we want this parsed different in loose mode?

Reply:
Loose mode just means you aren’t compliant CSS, right? It doesn’t mean that this:

https://github.com/shellscape/postcss-values-parser/pull/45/files#diff-cc37320af8aeb8b7582dad90cc114e55R219

```
-16px -1px -1px 16px
```

Should be parsed as 3 operators and 4 numbers, it should be four numbers.

But, I mostly leave that question to you as the maintainer. Did I misunderstand that test?

Reply:
@jwilsson please weigh in here. I recall that we implemented loose mode specifically for less and scss. Will removing how this is processed in loose mode have negative consequences?

Reply:
Good to know. This might require a major version, as it's going to break some expected functionality.

Reply:
Yeah, the idea behind `loose` mode was to target CSS-like languages (such as Less/SCSS). 

I definitely think we should tag a major after this one is merged, I know that at least Prettier is using this library (and that's a pretty large project) so we probably wouldn't want to change the behaviour unexpectedly.

Reply:
Lemme see if I can loop those folks into this. I'll have to comb through the commits to figure out who that was, but we'll get this in once we get some more eyes on it.

Reply:
@evilebottnawi can you comment as to whether or not this will impact prettier?